crux/src/Crux/Goal.hs

```diff
@@ -265,11 +266,11 @@ proveGoalsOffline adapters opts ctx explainFailure (Just gs0) = do
             let vals = evalModelFromEvents evs
             explain <- explainFailure (Just evalFn) p
             let locs = map eventLoc evs
-            return (locs, NotProved explain (Just (vals,evs)))
+            return (locs, NotProved explain (Just (vals,evs)) [])
```

Comment:
I believe we decided to do it for online solving only in the first iteration. I don't remember why, maybe @travitch can correct me if I'm wrong?

Reply:
I figured we can at least start in online mode, but it also wasn't obvious to me that we *could* in offline mode. Don't we want to inspect the result to figure out if we even want to ask for abducts? We don't have a persistent solver connection.  I guess we could just ask for N when constructing our initial query, but how does CVC5 handle `get-abduct` request N if N-1 failed because there are no more abducts?

Reply:
We would "fail" by timing out on the `get-abduct` query, so if it timed out on abduct N-1, it would most likely time out on abduct N as well. We would need to solve that problem in online-mode as well - by having individual timeouts, and stop asking when we reach a single timeout. I suppose this  would be slightly different in offline mode.